### DISCUSSION TITLE
Extracting embplant_pt from the assemblies failed.

### DISCUSSION THREAD

**Discussion by Razi1026** [Kinggerm/GetOrganelle#219] [Q&A]
Hello,

I ran this command for plants and after 3 days, my job failed:

get_organelle_from_reads.py -1 R1.fastq -2 R2.fastq -t 1 -o cochlearia.plastome -F embplant_pt -R 15 -k 21,45,65,85,105 --reduce-reads-for-coverage inf --max-reads inf

Extracting embplant_pt from the assemblies failed!!!

How can I solve this? Thanks!

[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/10143810/get_org.log.txt)

**Comment by Kinggerm** [comment]
`--reduce-reads-for-coverage inf --max-reads inf` is inputting way too many reads in your case!

Your `Estimated embplant_pt-hitting base-coverage` is as high as 17905.68x.
Please see the instruction of these arguments.

**Comment by Razi1026** [comment]
Thanks for the quick reply!

so do you think if I run the command without them, it will be better to work by default?
`get_organelle_from_reads.py -1 R1.fastq -2 R2.fastq -o cochlearia.plastome -F embplant_pt -R 15 -k 21,45,65,85,105 -t 1 `

I appreciate it if you had any suggestions for my case. Thanks!

**Reply by Kinggerm** [reply]
I'm not sure what arguments work best for you. You have to fine-tune them according to the results. Please refer to the log file and https://github.com/Kinggerm/GetOrganelle/wiki/FAQ#what-should-i-do-with-incomplete-resultbroken-assembly-graph

**Reply by Razi1026** [reply]
Thanks! I ran without `--reduce-reads-for-coverage inf --max-reads inf` and I got the result.

I have the same issue with plant mitochondria as well: 
`get_organelle_from_reads.py -1 R1.fastq -2 R2.fastq -o cochlearia.mito -F embplant_mt -R 15 -k 21,45,65,85,105`

I ran with ` -R 20 -P 1000000 ` as well for a run and I removed `--reduce-reads-for-coverage inf --max-reads inf` . But all runs failed to extract mitochondrial sequence after long days.

This time I try to run with `-R 30 -k 21,45,65,85,105  --reduce-reads-for-coverage 2000` to see!

I appreciate it if you have any suggestions to save time. Thanks!
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/10186344/get_org.log.txt)

**Reply by Kinggerm** [reply]
Please try to increase the word size. It seems that it was extending too many other reads.

Besides, please keep in mind that most people will just get contigs for the plant mitochondrial genome because of its complex problem. But you are getting nothing so there is definitely room to improve.

**Reply by Razi1026** [reply]
Thanks. I ran again with -w 100 and it worked. but I think there is more reads that I am not sure they are from mitochondria. It had 10 scaffolds and one of them 60% match with mitochondria sequence according with NCBI. 

Do you have any suggestion for this problem? Many thanks!
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/10251330/get_org.log.txt)

**Reply by Kinggerm** [reply]
Incomplete results (scaffolds) may be error-prone (see [GetOrganelle paper](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02154-5)).

I would recommend finetuning the parameters again for the circular result. A very important and helpful step is looking at the assembly graph. You may search for similar posted issues/discussions to see how others solve their problem.
